services/fabricv4/docs/BulkPortRequest.md

```diff
@@ -4,7 +4,7 @@
 
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**Data** | Pointer to [**[]Port**](Port.md) |  | [optional] 
+**Data** | Pointer to [**[]PortRequest**](PortRequest.md) |  | [optional] 
```

Comment:
This is a docs-only change; you can see [the model is already using `[]PortRequest` in released code](https://github.com/equinix/equinix-sdk-go/blob/abd895736e6dde2b475a74525ea883acc18af97b/services/fabricv4/model_bulk_port_request.go#L19).